### ISSUE TITLE
bgn-binary+keys doesn't have any keys?

### ISSUE THREAD

**Issue by atwok** [Zsh-Packages/fzf#2]
❯ zinit pack"bgn-binary+keys" for fzf
Downloading package.json for fzf
Parsing package.json...
Found the profile `bgn-binary+keys'

WARNING: the profile uses dl'' ice however there's no z-a-patch-dl annex loaded (the ice will be inactive, i.e.: no additional files will be downloaded)

Version: 0.19.0

Downloading junegunn/fzf-bin...
(Requesting `fzf-0.20.0-linux_amd64.tgz'...)
ziextract: Unpacking the files from: `fzf-0.20.0-linux_amd64.tgz'...
ziextract: Successfully extracted and assigned +x chmod to the file: `fzf'.
bin-gem-node annex: Created the fzf shim and set +x on the fzf binary
.zinit-load-plugin:source:95: no such file or directory: /home/atwok/.zinit/plugins/fzf/key-bindings.zsh

**Comment by psprint** [comment]
You need `zinit light zinit-zsh/z-a-patch-dl` loaded before like the warning says.

Now, after the `fzf` plugin has been installed, you can `zinit update fzf` as soon as you load the annex to fix the problem.

**Comment by psprint** [comment]
I've now also extended Zinit so that package's profile can **require** the Patch-Dl annex. After `zinit self-update` the command:

```zsh
zinit pack"bgn-binary+keys" for fzf
```

will not succeed unless there is the Patch-Dl annex loaded.

**Comment by atwok** [comment]
Ok, so I've done this.  And I can see key_bindings.zsh and _fzf_completion sitting along with the fzf binary, but when I do ** on my command line, fzf isn't automagically launching.  I'm not sure why that is.

**Comment by nekeal** [comment]
@atwok You are completely right, I've encountered the same problem today. The reason for that is that the `completion.zsh` file isn't sourced. If you source it manually, completions start working. 

@psprint I'm not sure if this is good approach but even fzf's install script adds following line to the startup script:

`[[ $- == *i* ]] && source "/home/test-zsh-user/.fzf/shell/completion.zsh" 2> /dev/null`
